pkg/render/dex_utils.go

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) 2019 Tigera, Inc. All rights reserved.
```

Comment:
Could we please have this named something other than dex_utils please. You could just name the dex_config.